# -------------------- 神经网络参数 --------------------
yolo_name: "yolov8"
classify_model: "/home/yeltsa/Desktop/Omni_vision/src/omni/assets/tiny_resnet.onnx"
yolov8_model_path: "/home/yeltsa/Desktop/Omni_vision/src/omni/assets/yolov8.xml"
device: "CPU"

threshold: 0.5               # double
max_angle_error: 30.0        # double, degree
min_lightbar_ratio: 0.1      # double
max_lightbar_ratio: 0.5      # double
min_lightbar_length: 5.0     # double
min_armor_ratio: 1.0         # double
max_armor_ratio: 5.0         # double
max_side_ratio: 0.2          # double
min_confidence: 0.8          # double
max_rectangular_error: 15.0  # double, degree

# -------------------- ROI --------------------
use_roi: false
roi:
  x: 0          # int
  y: 0          # int
  width: 640    # int
  height: 480   # int

# -------------------- 相机参数 --------------------
camera:
  id: 0
  fov_x: 70.0
  fov_y: 42.0
  cam_yaw_offset: 180.0
